Blackfynn/data-portal
# Mailing list link and contact us fixes #1864pn

# Description

The purpose of this PR is to add a mailing list link and contact us dialog fixes.

## Type of change


# How Has This Been Tested?

- Click on "Contact us" in the footer
- Dialog should be responsive
- Dismiss dialog
- Click on "Contact us" in the footer
- Contact us should open
- Dismiss dialog
- Click on "Subscribe to mailing list" in the footer
- Should be taken to the subscription form in a new tab

# Checklist:
